lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -496,7 +502,7 @@ dictionary OnchainPaymentLimitsResponse {
 [Enum]
 interface PayAmount {
     Bitcoin(u64 receiver_amount_sat);
-    Asset(string asset_id, u64 receiver_amount);
+    Asset(string asset_id, f64 receiver_amount);
```

Comment:
Have you considered avoiding floating point overall? IMO it would be safer to always rely on the integer amount of the smallest unit of the asset.

Reply:
I considered it and would also prefer to avoid floats. But the smallest unit of an asset is 1 sat, so to send 1.50 USDt you would need to set 150000000. I could see implementors being confused by this. What do you think @roeierez ?

Reply:
I think we don't have a choice here. The user should be able to set a non integer number of the native asset. The alternative of exposing the precision to the app and expecting the app to do the calculation do no good to the UX and still doesn't avoid the conversion.